src/core_modules/capture-core/utils/converters/date/stringToMomentDateFormat.js

```diff
@@ -11,6 +11,6 @@ import { systemSettingsStore } from '../../../metaDataMemoryStores';
 export function convertStringToDateFormat(date: string) {
```

Comment:
With this change, I think a more suitable name for this function is `convertStringToServerFormat`. The function's comment definition also needs to be updated. WDYT?